extra_data/export.py

```diff
@@ -242,7 +255,31 @@ def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
     streamer.start()
     for tid, train_data in data.trains():
         if train_data:
-            streamer.feed(train_data)
+            if append_detector_modules:
+                if source_glob == '*':
+                    warn(' You are trying to stack detector-module sources \
+with a global windcard (\'*\'). If there are non-detector sources in your \
+run, this will fail.\n')
+                stacked = stack_detector_data(train_data, 'image.data')
+                merged_data = {}
+                # the data key pretends this is online data from SPB
+                merged_data['SPB_DET_AGIPD1M-1/CAL/APPEND_CORRECTED'] = {
+                    'image.data': stacked
+                    # add other data-set objects (by key) here?
+                }
+                # add other sources (by key) here?
+                sec, frac = gen_time() # use time-stamps from file data?
+                metadata = {}
+                metadata['SPB_DET_AGIPD1M-1/CAL/APPEND_CORRECTED'] = {
+                    'source': 'SPB_DET_AGIPD1M-1/CAL/APPEND_CORRECTED',
+                    'timestamp.tid': tid,
+                    'timestamp.sec': sec,
+                    'timestamp.frac': frac
+                }
```

Comment:
I've just noticed that the generated timestamps for other sources are conditional on a `dummy_timestamps=True` parameter to `ZMQStreamer`. I think we should keep that consistent, and not generate timestamps here.

Possibly that means that the `karabo-bridge-serve-files` command needs a `--dummy-timestamps` option as well so that it can produce data suitable for OnDA.

Reply:
Yes, that's doable